modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.client/src/main/scala/edu/gemini/seqexec/web/client/components/sequence/HeadersSideBar.scala

```diff
@@ -33,30 +35,37 @@ object HeadersSideBar {
   }
 
   class Backend(val $: BackendScope[Props, State]) extends TimerSupport {
+    def updateOperator(name: String): Callback =
+      $.props >>= { p => Callback.when(p.isLogged)(Callback(SeqexecCircuit.dispatch(UpdateOperator(name)))) }
+
     def updateState(value: String): Callback =
       $.modState(_.copy(currentText = Some(value)))
 
+    def setupTimer: Callback =
+      // Every 2 seconds check if the field has changed and submit
+      setInterval(submitIfChanged, 2.second)
+
     def submitIfChanged: Callback =
       ($.state zip $.props) >>= {
-        case (s, p) => Callback.empty // Callback.when(s.currentText =/= p.operator())(Callback.empty) // We are not submitting until the backend properly update this property
+        case (s, p) => Callback.when(s.currentText =/= p.model().operator)(updateOperator(~s.currentText))
```

Comment:
It is the same as `orZero` or if you prefer `getOrElse("")`. I'm not sure if it is the preferred syntax these days but I learned it some time ago